src/main/java/mate/academy/intro/controller/BookController.java

```diff
@@ -17,41 +20,50 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/books")
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books", description = "Get a list of all available books")
+    public List<BookDto> getAll(Pageable pageable) {

```

Comment:
```suggestion
    public List<BookDto> getAll(@ParameterObject @PageableDefault Pageable pageable) {

```
Read about these annotations